community/openebs/README.md

```diff
@@ -0,0 +1,148 @@
+# OpenEBS
+
+**It is based on Helm community chart [openebs](https://github.com/openebs/openebs/tree/master/k8s/charts/openebs)**
+
+[OpenEBS](http://openebs.io/) is a cloud-native storage solution built with the goal of providing containerized storage for containers. Using OpenEBS, a developer can seamlessly get persistent storage for stateful applications on Kubernetes with ease.
+
+## Introduction
+
+This chart bootstraps a [OpenEBS](https://github.com/openebs/openebs) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+

```

Comment:
Added Image Security Enforcement notice below

Reply:
Thanks for adding this, @muratkars. My mistake, I didn't quite use the right terminology in my previous comment. What you'd want to ask a user to do is add the repository on dockerhub where your images live, as opposed to adding the entire registry. I would suggest that in the text you have there, telling the user that they'll need to add `docker.io/openebs/*` to their image policy, so that they don't have to look up the location in values.yaml